src/studying.md

```diff
@@ -133,6 +133,20 @@ answer. Pressing the <kbd>spacebar</kbd> or <kbd>Enter</kbd> will select **Good*
 See [Deck Options](deck-options.md) and the [FAQ](https://faqs.ankiweb.net/what-spaced-repetition-algorithm.html)
 to learn more about how the algorithm works.
 
+## How to choose the answer button
+
+The sections above explain *what* the answer buttons do, but you may also want to know *how* to choose an answer button intuitively,
+without thinking about the algorithm.
+
+- **Again** should be pressed if your answer is incorrect. If your answer is partially correct, you should be strict with yourself: if it would count as
+a fail outside of Anki, then it counts as a fail in Anki as well.
+
+- **Hard** should be pressed if your answer is correct, but you had doubts about it or if you made a minor mistake. If your answer is 90% correct, press **Hard**.
```

Comment:
That's an Again. Minor mistake = mistake.

Reply:
No, that's Hard, that's why I said "minor"

Reply:
Again tanks the memory stability. If I made a non-critical mistake, it doesn't make sense to decrease memory stability by 1-2 orders of magnitude. So Hard, which doesn't decrease the interval length, makes more sense for something that is far closer to "fully recalled" than to "fully forgotten".

@dae this might seem like it contradicts my "no references to interval lengths or anything algorithmic in order to make the guidelines beginner-friendly" rule. But that rule is only for what is **written**. We, the people writing the guidelines, are free to use any reasoning behind the scenes, so long as what the manual says in the end is beginner-friendly.

Reply:
That only makes sense if you assume the information to be recalled is a unitary memory. I can quite reasonably think of the situation as complete forgetting of one of the component required to arrive at a correct answer.

Also, the example I shared before: If I see 春日 I think はるひ which is the correct reading but I always seem to forget that another possible reading is かすが、if I think "ah, this is a minor lapse of memory" I'm never going to unlearn my habit. So, I think Again is the right choice.